buildchow/templates/core-jobs.yaml

```diff
@@ -5,9 +5,11 @@
       master
     devstudio-stream:
       master
-    target-platform-version: 4.50.0.Beta1-SNAPSHOT
-    target-platform-version-maximum: 4.50.0.Beta1-SNAPSHOT
-    disabled: true
+    target-platform-version: 4.50.0.CR1-SNAPSHOT
+    target-platform-version-maximum: 4.50.0.CR1-SNAPSHOT
+    blockBuildWhenDownstreamBuilding: false
```

Comment:
In future we may have component jobs interconnected. For example... 

https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/jbosstools-javaee_master/ -> https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/jbosstools-javaee-tests-matrix_master/

So, the property makes sense, unless we decide we don't want tests to run without first rebuilding the plugins/features?

And yes, it's possible to handle the linkages. I've already set up a few, using two different Jenkins plugins (downstream-ext and trigger-parameterized-builds). It just means more deviations from the standard template.